storage/postgres/postgres_test.go

```diff
@@ -32,7 +31,7 @@ func TestReadAuthorizationModelPostgresSpecificCases(t *testing.T) {
 	ctx := context.Background()
 	store := "store"
 	modelID := "foo"
-	schemaVersion := 4 // which is not an enum value... yet.
+	schemaVersion := "7.8"
 
 	bytes, err := proto.Marshal(&openfgapb.TypeDefinition{Type: "document"})
```

Comment:
We use `protojson.Marshal` in some places and this uses `proto.Marshal` in others. We should confirm they are identical and strive to use only one. We can save that for a future PR, but let's make sure and add an issue to verify and refactor.

Reply:
Addressed. Thanks!